doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+"""
+========================================================================
+Reconstruction of the diffusion signal with the correlation tensor model
+========================================================================
+
+The Correlation Tensor Model (CTI) is an advanced extension of the Diffusion Kurtosis Model (see :ref:sphx_glr_examples_built_reconstruction_reconst_dki.py). [ not sure]
```

Comment:
Write instead: The Correlation Tensor MRI (CTI) is a method that uses double diffusion encoding data to resolve sources of kurtosis. Compared to the  Q-space Trajectory Imaging (see :ref:sphx_glr_examples_built_reconstruction_reconst_qti.py) it does not assume that tissues can be represented by a sum of multiple Gaussian compartments.

Reply:
Here we also have to add the reference to the paper (Henriques et al. Neuroimage 2020). See how references are being done in other DIPY tutorials.